modules/python3/pythoninterpreter.cpp

```diff
@@ -59,11 +57,11 @@ PYBIND11_EMBEDDED_MODULE(inviwo_internal, m) {
     });
 }
 
-void PythonInterpreter::init(Python3Module* module) {
+PythonInterpreter::PythonInterpreter(Python3Module* module) : isInit_(false) {
     namespace py = pybind11;
 
     if (isInit_) {
-        throw Exception("Python already initialized", IvwContext);
+        throw ModuleInitException("Python already initialized", IvwContext);
     }
```

Comment:
The ´if (isInit_)´ test should not be needed anymore since this code is now in the constructor.

Reply:
Unless someone tries to create annother pythoninterpreter...